cmd/migrate.go

```diff
@@ -59,20 +63,30 @@ func migrateCmd() *cobra.Command {
 				return nil
 			}
 
+			backfillConfig := backfill.NewConfig(
+				backfill.WithBatchSize(flags.BackfillBatchSize()),
+				backfill.WithBatchDelay(flags.BackfillBatchDelay()),
+			)
+
 			// Run all migrations after the latest version up to the final migration,
 			// completing each one.
 			for _, mig := range migs[:len(migs)-1] {
-				if err := runMigration(ctx, m, mig, true); err != nil {
+				if err := runMigration(ctx, m, mig, true, backfillConfig); err != nil {
 					return fmt.Errorf("failed to run migration file %q: %w", mig.Name, err)
 				}
 			}
 
 			// Run the final migration, completing it only if requested.
-			return runMigration(ctx, m, migs[len(migs)-1], complete)
+			return runMigration(ctx, m, migs[len(migs)-1], complete, backfillConfig)
 		},
 	}
 
+	migrateCmd.PersistentFlags().Int("backfill-batch-size", backfill.DefaultBatchSize, "Number of rows backfilled in each batch")
+	migrateCmd.PersistentFlags().Duration("backfill-batch-delay", backfill.DefaultDelay, "Duration of delay between batch backfills (eg. 1s, 1000ms)")
```

Comment:
These don't need to be persistent flags (they don't apply to subcommands, `migrate` doesn't have any). use `.Flags()` instead, like the `--complete` flag.

This might explain why the `cli-description.json` didn't notice these flags.